spec/ics-007-tendermint-client/README.md

```diff
@@ -41,18 +41,22 @@ The Tendermint client state tracks the current validator set, latest height, and
 ```typescript
 interface ClientState {
   validatorSet: List<Pair<Address, uint64>>
+  trustingPeriod: uint64
+  unbondingPeriod: uint64
```

Comment:
do we need to check that ubd > trusting period?

Reply:
Sure, we can.